lib/srv/db/cloud/watchers/rds.go

```diff
@@ -266,26 +284,60 @@ func (f *rdsAuroraClustersFetcher) String() string {
 		f.cfg.Region, f.cfg.Labels)
 }
 
-// rdsFilters returns filters to make sure DescribeDBInstances call returns
+// rdsInstanceEngines returns engines to make sure DescribeDBInstances call returns
 // only databases with engines Teleport supports.
-func rdsFilters() []*rds.Filter {
-	return []*rds.Filter{{
-		Name: aws.String("engine"),
-		Values: aws.StringSlice([]string{
-			services.RDSEnginePostgres,
-			services.RDSEngineMySQL,
-			services.RDSEngineMariaDB}),
-	}}
+func rdsInstanceEngines() []string {
+	return []string{
+		services.RDSEnginePostgres,
+		services.RDSEngineMySQL,
+		services.RDSEngineMariaDB,
+	}
```

Comment:
nitpick: these probably can be just `var rdsInstanceEngines = []string{...}`?

Reply:
I thought the same, but I wanted to make it const which is unfortunately not possible in Go. I think mutable globals are a code smell